Makefile

```
@@ -450,6 +451,7 @@ compress-tar:
 .PHONY: ci-env-up
 ci-env-up:
 	@$(call func_echo_status, "$@ -> [ Start ]")
+	touch $(OTEL_CONFIG); chmod 777 $(OTEL_CONFIG)
```

```suggestion
	touch $(OTEL_CONFIG)
	chmod 777 $(OTEL_CONFIG)
```
One cmd for one thing.